packages/suite/src/utils/suite/tests/homescreen.test.ts

```diff
@@ -0,0 +1,109 @@
+import * as Canvas from 'canvas';
+import * as crypto from 'crypto';
+
+import * as homescreen from '../homescreen';
+
+const homescreensPath = '../suite-web/static/images/suite/homescreens';
+
+// to simplify assertions of hex return values
+const getHash = (str: string) => {
+    return crypto
+        .createHash('md5')
+        .update(str)
+        .digest('hex');
+};
+
+const getMockElementToImageData = (image: HTMLImageElement) => {
+    // @ts-ignore
+    return (element, w, h) => {
+        const canvas = Canvas.createCanvas(w, h);
+        const ctx = canvas.getContext('2d');
+        ctx.drawImage(image, 0, 0);
+
+        // you can test visually if proper imageData was returned this way:
+        // console.log(`<img src="${canvas.toDataURL()}" />`);
+
+        const imageData = ctx.getImageData(0, 0, w, h);
+        return imageData;
+    };
+};
+const imgHashFixtures = [
+    {
+        model: 2,
+        img: 'btc.png',
+        hexHash: '0f15d6b5d66126e1c9fc54c263323bf9',
+    },
+    {
+        model: 2,
+        img: 'doge.png',
+        hexHash: 'cfae6996e66d3d67dbfe7cbe01805dd5',
+    },
+    {
+        model: 1,
+        img: 'ancap.png',
+        hexHash: '28ec73580bab3dfcc1e827caca49d0cd',
+    },
+    {
+        model: 1,
+        img: 'anonymous.png',
+        hexHash: 'f82d789c632b6924486c80d75c53285f',
+    },
+] as const;
+
+interface Fixture {
+    img: typeof imgHashFixtures[number]['img'];
+    model: typeof imgHashFixtures[number]['model'];
+    hexHash: typeof imgHashFixtures[number]['hexHash'];
+}
+
+describe('homescreen', () => {
+    describe('imageDataToHomescreen', () => {
```

Comment:
should be "it" or "test"

Reply:
Depends on what you want. Here I want to organize tests in 3 levels deep structure like this

![image](https://user-images.githubusercontent.com/30367552/62924902-6b62aa00-bdb1-11e9-9f4e-afb7b5840b7c.png)

This way it is used on many other places, for example: src/utils/suite/tests/validators.test.ts, src/utils/suite/tests/router.test.ts ...